arches_for_science/media/js/views/components/workflows/project-report-workflow/download-report.js

```diff
@@ -37,6 +46,24 @@ define([
         getProjectName();
 
         const generateReport = async(template) => {
+            const relatedObjects = await getRelatedResources(projectId);
+            const collections = relatedObjects.related_resources.filter(rr => rr.graph_id == collectionGraphId);
+            const allPhysicalThingsResponse = collections.map(async(collection) => {
+                const collectionRelatedResources = await getRelatedResources(collection.resourceinstanceid);
+                return collectionRelatedResources?.related_resources.filter(rr => rr.graph_id == physicalThingGraphId);
+            });
+
+            self.physicalThings = [].concat(...(await Promise.all(allPhysicalThingsResponse))).filter(res => {
```

Comment:
Tried a couple projects, and generating the reports all failed like this on the last step. Let me know if you see the same thing.
```py
2023-09-21 15:07:13 django.core.exceptions.ValidationError: ['“” is not a valid UUID.']
2023-09-21 15:07:13 [21/Sep/2023 19:07:13] "GET /api/bulk_disambiguated_resource_instance?v=beta&resource_ids= HTTP/1.1" 500 190147
```

Reply:
I don't see this happening... hmmm. not sure what causes the issue.
Maybe you have something that I don't or vice versa.

Reply:
Aha! I was able to reproduce the same problem on dev/1.1.x, so it's not on this branch. 👍 

My issue was I didn't have any observations. How should I create some?